lib/rex/ui/text/shell.rb

```diff
@@ -178,6 +178,9 @@ def run(&block)
           input.prompt.gsub!(/%J/, framework.jobs.length.to_s)
           input.prompt.gsub!(/%L/, Rex::Socket.source_address("50.50.50.50"))
           input.prompt.gsub!(/%D/, ::Dir.getwd)
+          if framework.db && framework.db.workspace
```

Comment:
Fixed in https://github.com/rapid7/metasploit-framework/commit/5f8d58f2140eabad7b674bea40ab709d63fad1d9.